app/Blog.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Blog extends Model
+{
+    //
```

Comment:
why have this comment without content ? if not comment please remove

Reply:
i fixed